subsys/net/conn_mgr/events_handler.c

```diff
@@ -24,7 +24,7 @@ static void conn_mgr_iface_events_handler(struct net_mgmt_event_callback *cb,
 {
 	int idx;
 
-	NET_DBG("Iface event %u received on iface %d (%p)", mgmt_event,
+	NET_DBG("%s event %x received on iface %d (%p)", "Iface", mgmt_event,
```

Comment:
You missed the `0x%x` change here and the two changes below. The reason why I am asking this is that previously the output was decimal but now the print is in hex and people might not realize that because the hex might look like dec.

Reply:
sorry, it got dropped in the save rebase cycle, should be fixed